portfolio/src/main/webapp/styles/comments.css

```diff
@@ -265,7 +265,7 @@ main {
 
 .comment-content {
     position: relative;
-    width: 100%;
+    flex-grow: 1;
```

Comment:
Changes in this file again are not about the map section, so move them to a different commit